core/server/master/src/test/java/alluxio/master/block/BlockMasterTest.java

```diff
@@ -82,6 +99,15 @@ public class BlockMasterTest {
   private static final Map<Block.BlockLocation, List<Long>> NO_BLOCKS_ON_LOCATION
       = ImmutableMap.of();
   private static final Map<String, StorageList> NO_LOST_STORAGE = ImmutableMap.of();
+  public static final Map<String, List<String>> LOST_STORAGE =
+          ImmutableMap.of(MEM.toString(), ImmutableList.of());
```

Comment:
You can also use "MEM" here to avoid dependent to stress-shell.

Reply:
Actually I also needed the `RegisterStreamer` from `alluxio-core-server-worker`, which is brought in by `alluxio-stress-shell`.